<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Change_student</class>
 <widget class="QDialog" name="Change_student">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change</string>
  </property>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>drink</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">inter_buttons</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>man</string>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">sex_buttons</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>listen</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">inter_buttons</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>study</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">inter_buttons</string>
   </attribute>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Chinese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maths</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>history</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>103</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>48</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>id          </string>
   </property>
  </widget>
  <widget class="QLabel" name="subject">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>53</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>subject </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>play</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">inter_buttons</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>30</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="age">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>131</y>
     <width>53</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>age        </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>131</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>woman</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">sex_buttons</string>
   </attribute>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="interest">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>interest</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_next">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>200</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>eat</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">inter_buttons</string>
   </attribute>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>26</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_change">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>109</x>
     <y>30</y>
     <width>52</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>name    </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="sex">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>103</y>
     <width>23</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string>sex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>happy</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">inter_buttons</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="inter_buttons">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="sex_buttons"/>
 </buttongroups>
</ui>
